components/ai_chat/resources/page/state/active_chat_provider.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
Should this file be named `ActiveChatContext` to be consistent and be obvious it contains a React Context, Context Provider and context state type?